### ISSUE TITLE
libafl_jumper cargo test --no-default-features leads to SIGSEGV

### ISSUE THREAD

**Issue by Marcondiro** [AFLplusplus/LibAFL#2629]

**Comment by domenukk** [comment]
Yeah the no-std feature is low-key defunct. I thought I could make it work but it turns out there's a reason ppl use a libc...
Maybe we should remove the feature flags in that one?

**Comment by Marcondiro** [comment]
Got it, I don't know if it is better to remove the feature or put everything that still works behind the `std` feature ?

**Comment by domenukk** [comment]
Right now std works (and that's the default)
